rmw_opendds_shared_cpp/src/OpenDDSNode.cpp

```diff
@@ -0,0 +1,241 @@
+// Copyright 2015-2017 Open Source Robotics Foundation, Inc.
```

Comment:
Maybe rename the file to `opendds_node.cpp`?

Reply:
See comment in opendds_node.cpp.